plotting/plot_coords.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
```

Comment:
Add a doc str

Reply:
> @jdha All the changes look sensible by eye. Can you state which test I should conduct? I will also then check it's interaction with tide options (I expect no interaction). Then it is good to go.

All I've done so far is to run the benchmark extraction for ln_tra, ln_dyn2d and ln_dyn3d and combinations thereof. 
Output hasn't been verified yet. Plan is to generate a synthetic velocity field rotated at 45\deg and magnitude 2**0.5 to produce and u and v field of 1m/s - which could then be used as one of the unit tests.